extensions/scanning/tests/scanning_collection.test.js

```diff
@@ -0,0 +1,32 @@
+import o from 'ospec';
+import Ajv from 'ajv';
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
+import { promises as fs } from 'fs';
+import { AjvOptions, DefaultTimeoutMillis } from '../../../validation.js';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const schemaPath = join(__dirname, '..', 'schema.json');
+const examplePath = join(__dirname, '..', 'examples/collection.json');
+
+o.spec('Scanning Extension Collection', () => {
+  o.specTimeout(DefaultTimeoutMillis);
+  let validate;
+  const ajv = new Ajv(AjvOptions);
+
+  o.before(async () => {
+    const data = JSON.parse(await fs.readFile(schemaPath));
+    validate = await ajv.compileAsync(data);
+  });
+
+  o('Example should pass validation', async () => {
```

Comment:
Add a test where the validation fails

Reply:
It looks like the only thing the schema change does is make sure you can put the `stac_extensions` link to this extension in a collection JSON file. So I'm not sure there's anything which could be tested: if you remove the link, this extension doesn't apply anymore. I guess you could set the `type` to `Catalog`, but that's just saying "we don't support this yet" rather than "we've implemented a new check".

Reply:
We plan to add some rules for validation around summaries for these extension schemas, probably later in this sprint. I think we could add some failing tests then?

Reply:
You're right @l0b0 , my idea was to remove the type but it is not a very relevant test.